FlowersOrder/FlowersDetailScreen/FlowersDetailViewModel.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  FlowersDetailViewModel.swift
+//  FlowersOrder
+//
+//  Created by Dragos Cosma on 06/02/2020.
+//  Copyright © 2020 NewBlack. All rights reserved.
+//
+
+import Foundation
+
+protocol FlowersDetailViewModelProtocol {
+    var description: String { get }
+    var price: String { get }
+    var delivered_to: String { get }
+    var image_string: String { get }
+}
+
+class FlowersDetailViewModelImplementation: FlowersDetailViewModelProtocol {
+
+    let description: String
+    let price: String
+    let delivered_to: String
```

Comment:
I think you should use camel case names here too, to be consistent with the rest of the app (and the Swift guidelines in general).